app/services/imports/logs_import_service.rb

```diff
@@ -72,7 +72,7 @@ def safe_string_as_decimal(xml_doc, attribute)
       if str.nil?
         nil
       else
-        BigDecimal(str, exception: false).round(2)
+        BigDecimal(str, exception: false)&.round(2)
```

Comment:
this is an unrelated change but as it's on theme with nil safeness I thought I'd include it, it's for another migration log import failure I saw

Reply:
Can we add a failing test for this?

Reply:
We sure can